modules/saku-policy-store/src/saku/routes.clj

```diff
@@ -0,0 +1,234 @@
+              :policies   result}}))
+
+(defn retract-statements-handler
+  [request]
+  (let [{:saku/keys [input-data ctx]} request
+        result (dal/retract-statements {:dal-obj (:dal-obj ctx)} input-data)]
+    {:status 200
+     :body   result}))
+
+(defn describe-policies-handler
+  [request]
+  (let [{:saku/keys [input-data ctx]} request
+        result (dal/get-policies {:dal-obj (:dal-obj ctx)} input-data)]
+    {:status 200
+     :body   {:policies result}}))
+
+(defn ->evaluate-policy
+  [policy]
+  (-> policy
+    (update :policy/statements (fn [statements]
+                                 (mapv (fn [statement]
+                                         (-> statement
+                                           (sets/rename-keys {:statement/actions    :actions
+                                                              :statement/effect     :effect
+                                                              :statement/identities :identities
+                                                              :statement/resources  :resources})
+                                           (update :effect #(-> % :effect csk/->SCREAMING_SNAKE_CASE_KEYWORD))))
+                                   statements)))
+    (sets/rename-keys {:policy/drn        :drn
+                       :policy/statements :statements})))
+
+(defn evaluate-many
+  [{:keys [dal-obj]} evaluate-many-input]
+  (let [{:keys [drns identities actionId]} evaluate-many-input
+        db (dal/db dal-obj)
+        ctx {:dal-obj dal-obj :db db}
+        resource-policies (mapv ->evaluate-policy (dal/get-policies ctx {:drns drns :policyType "Resource"}))
+        drn->resource-policy (into {} (map (juxt :drn identity)) resource-policies)
+        identity-policies (mapv ->evaluate-policy (dal/get-policies ctx {:drns identities :policyType "Identity"}))
+        drn->identity-policy (into {} (map (juxt :drn identity)) identity-policies)]
+    (into []
+      (map (fn [drn]
+             (let [resource-policy (drn->resource-policy drn)
+                   evaluate-argm {:drn                   drn
+                                  :action                actionId
+                                  :resource-policy       resource-policy
+                                  :identity-policies-map drn->identity-policy}
+                   result (core/evaluate-one evaluate-argm)]
+               (log/info (assoc evaluate-argm
+                           :result result
+                           :msg "Evaluating effect..."))
+               {:drn drn :result result})))
+      drns)))
+
+(defn evaluate-one-handler
+  [request]
+  (let [{:saku/keys [ctx]
+         :keys      [parameters]} request
+        many-result (evaluate-many {:dal-obj (:dal-obj ctx)}
+                      (-> (:body parameters)
+                        (update :drn vector)
+                        (sets/rename-keys {:drn :drns})))
+        result (-> many-result first :result)]
```

Comment:
Simplified the implementation of evaluate one since it was about 95% the same as many. All we need to do is get the first result off of many, and we get the same behavior.